demo.py

```diff
@@ -0,0 +1,12 @@
+from twitter.stream import StreamExecutor, GlobalStreamListener
+
+if __name__ == "__main__":
+    listener = GlobalStreamListener(lan="en")
+    executor = StreamExecutor(listener)
+    executor.loop()
+
+# geopy.exc.GeocoderServiceError: [Errno -2] Name or service not known
+# geopy.exc.GeocoderTimedOut: Service timed out
+# ssl.SSLError: [SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:2309)
+
```

Comment:
Are these important?

Reply:
It's hard to follow without comments, could you add some?

Reply:
no

Reply:
I ll remove them now

Reply:
> It's hard to follow without comments, could you add some?

I suppose you mean at datahandler.py, so I did now

Reply:
Good stuff! Thanks. I am downloading it now to see it in action.